README.md

```diff
@@ -19,13 +19,22 @@ This project enables caching in Spring-managed beans, by using Java 5 Annotation
 
 ## Usage ##
 
+Add the repository for the Momento snapshot version:
+
+    <repositories>
+      <repository>
+        <id>sonatype-snapshots</id>
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+      </repository>
+    </repositories>
```

Comment:
is there not a built-in for this? My maven-fu is very dated but I thought there were built-in repos for Maven Central and Maven Central Snapshots, so that you didn't have to put the URL in. Maybe that's not the case?

Reply:
I don't believe maven is configured to pull down shapshots from sonatype central by default.

Reply:
okay. I think I'm getting mixed up with gradle where there is like a prebuilt `mavenCentral()` thingy